
package za.co.telkom;

import javax.xml.ws.WebFault;


/**
 * This class was generated by Apache CXF 3.0.4
 * 2019-04-10T11:11:23.417+05:30
 * Generated source version: 3.0.4
 */

@WebFault(name = "authenticate3DSResultResponse", targetNamespace = "http://www.telkom.co.za")
public class Authenticate3DSResultSoapOut extends Exception {
    
    private za.co.telkom.Authenticate3DSResultResponse authenticate3DSResultResponse;

    public Authenticate3DSResultSoapOut() {
        super();
    }
    
    public Authenticate3DSResultSoapOut(String message) {
        super(message);
    }
    
    public Authenticate3DSResultSoapOut(String message, Throwable cause) {
        super(message, cause);
    }

    public Authenticate3DSResultSoapOut(String message, za.co.telkom.Authenticate3DSResultResponse authenticate3DSResultResponse) {
        super(message);
        this.authenticate3DSResultResponse = authenticate3DSResultResponse;
    }

    public Authenticate3DSResultSoapOut(String message, za.co.telkom.Authenticate3DSResultResponse authenticate3DSResultResponse, Throwable cause) {
        super(message, cause);
        this.authenticate3DSResultResponse = authenticate3DSResultResponse;
    }

    public za.co.telkom.Authenticate3DSResultResponse getFaultInfo() {
        return this.authenticate3DSResultResponse;
    }
}
